agir/events/views/event_views.py

```diff
@@ -549,6 +556,75 @@ def get_queryset(self):
         return Event.objects.upcoming(as_of=timezone.now(), published_only=False)
 
 
+@method_decorator(never_cache, name="get")
+class ConfirmEventGroupCoorganization(View):
+    def get(self, request, pk, *args, **kwargs):
+
+        if not self.request.user.is_authenticated:
+            return HttpResponseRedirect(reverse("dashboard"))
```

Comment:
Pour vérifier si la personne est connectée :
```suggestion
        if not self.request.user.is_authenticated or self.request.user.person is None:
            return HttpResponseRedirect(reverse("dashboard"))
```
Il peut y avoir des utilisateurs connectés qui ne sont pas des personnes (:alien: , :robot:,...)

Reply:
En réalité ce cas arrive vraiment ? Je pensais que le fait d'être authentifié suffisait

Reply:
Oui, je crois que ça peut arriver : en tout cas c'est comme ça qu'on vérifie partout dans le code